WTWSMS/events/WTWSMS_historical_events.txt

```diff
@@ -997,3 +997,76 @@ narrative_event = {
 		prestige = 150
     }
 }
+
+# 526 Antioch earthquake
+narrative_event = {
+	id = hist.24
+	title = hist.24.name
+	desc = hist.24.desc
+	picture = GFX_evt_falling_statue
+	border = GFX_event_narrative_frame_diplomacy
+	
+	only_playable = yes
+	
+	major = yes
+
+	trigger = {
+		year = 526
+		NOT = { has_global_flag = 526_antioch_earthquake }
+		b_antiocheia = {
+			holder_scope = { top_liege = { character = ROOT } }
+		}
+	}
+	
+    mean_time_to_happen = {
+        months = 3
+	}
+	
+    option = {
+		name = hist.24.a
+		
+		trigger = {
+			b_antiocheia = {
+				holder_scope = { top_liege = { character = ROOT } }
+			}
+		}
+		
+		b_antiocheia = {
+			destroy_random_building = yes
+			destroy_random_building = yes
+			destroy_random_building = yes
+			destroy_random_building = yes
+			destroy_random_building = yes
+			location = {
+				add_province_modifier = { name = horrible_earthquake duration = -1 }
+			}
+		}
+		set_global_flag = 526_antioch_earthquake
+	}
+	option = {
+		name = hist.24.b #we must send help
+		trigger = {
+			b_antiocheia = {
+				holder_scope = { top_liege = { character = ROOT } }
+			}
+		}
+		
+		b_antiocheia = {
+			destroy_random_building = yes
+			destroy_random_building = yes
+			location = {
+				add_province_modifier = { name = horrible_earthquake duration = -1 }
+			}
+		}
+		scaled_wealth = -0.25
+		piety = 200
+		set_global_flag = 526_antioch_earthquake
```

Comment:
Wouldn't it be better to set this flag and add the province modifier in an immediate block?

Reply:
Yes, it would.

Reply:
Yes, it won't appear in the tooltip.